p2p/peer_behaviour_test.go

```diff
@@ -0,0 +1,115 @@
+package p2p
```

Comment:
As far as I can see the tests below only use the public API 👍 If you want you can rename this test package here to `p2p_test`.